packages/v4/src/pages/home.js

```diff
@@ -0,0 +1,290 @@
+import React from 'react';
+import { Link } from 'theme-patternfly-org/components';
+import { SideNavLayout } from 'theme-patternfly-org/layouts';
+import {
+  Flex,
+  FlexItem,
+  Grid,
+  GridItem,
+  PageSection,
+  PageSectionVariants,
+  Title,
+  TextContent,
+  Text,
+  TextVariants
+} from '@patternfly/react-core';
+import { ArrowRightIcon } from '@patternfly/react-icons';
+import orb from '../images/pf4-logo.svg';
+import accessible from '../images/accessible.svg';
+import modular from '../images/modular.svg';
+import scale from '../images/scale.svg';
+import consistent from '../images/consistent.svg';
+import open from '../images/open.svg';
+import { AutoLinkHeader } from 'theme-patternfly-org/components/autoLinkHeader/autoLinkHeader';
+import './home.css';
+
+const aboutPatternFly = `Hello, we're PatternFly.`;
+const patternFlyPrinciples = 'Flyer fundamentals';
+
+export const HomePage = ({ location, layoutOptions }) => (
+  <SideNavLayout location={location} hasGdprBanner {...layoutOptions}>
+    <PageSection
+      variant={PageSectionVariants.light}
+      className="pf4-c-background-image ws-homepage-main-section"
+    >
+      <Grid>
+        <GridItem sm={12} md={8} mdOffset={2} lg={6} lgOffset={3} className="pf-u-py-2xl pf-u-text-align-center">
+          <TextContent>
+            <img src={orb} alt="PatternFly logo" className="fadeInDown animated fadeInOne" />
+            <Title headingLevel="h1" size="4xl" className="pf-m-white pf4-site-c-hero fadeIn animated fadeInTwo">
+              Build scalable experiences in the open.
+            </Title>
+            <Title size="xl" headingLevel="h2" className="pf-m-white pf-u-mb-md pf-u-mb-3xl-on-md fadeInUp animated fadeInThree">
```

Comment:
This seems more like text for a paragraph, not so much a heading. Does it make sense to be an h2?

Reply:
Changed to paragraph.